app/assets/javascripts/pretty-text/engines/discourse-markdown-it.js.es6

```diff
@@ -190,16 +199,24 @@ function setupImageDimensions(md) {
 }
 
 function renderAttachment(tokens, idx, options, env, slf) {
-  const linkOpenToken = tokens[idx];
-  const linkTextToken = tokens[idx + 1];
-  const split = linkTextToken.content.split("|");
-  const isValid = !linkOpenToken.attrs[
-    linkOpenToken.attrIndex("data-orig-href")
-  ];
-
-  if (isValid && split.length === 2 && split[1] === ATTACHMENT_CSS_CLASS) {
-    linkOpenToken.attrs.unshift(["class", split[1]]);
-    linkTextToken.content = split[0];
+  const linkToken = tokens[idx];
+  const textToken = tokens[idx + 1];
+
+  const split = textToken.content.split("|");
+  const content_split = [];
+
+  for (let i = 0, data; i < split.length; ++i) {
+    if (split[i] === ATTACHMENT_CSS_CLASS) {
```

Comment:
probably worth to have split[i] as a variable here as it's reused multiple times.

Reply:
Similar to the other place, it does not hold a specific value. It would have a generic name such as `part` or `fragment` which does not really add any value.